### PR TITLE
Fix requests of webapp

### PR DESCRIPTION
**Pull request by alandoni**

### CODE REVIEW

File: api/server.js [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
-
 console.log("Listening at port: " + app.get("port"));
 
-const routes = require('./routes');
-routes.set(app, mongo);
-
-app.get('/', (req, res) => {
-	res.sendfile(__dirname + '/dist/index.html');
-});
+//routes.set(app, mongo);
```
Comment (kiosia) [inline api/server.js]: nope.avi

File: api/server.js [hunk 2]
```diff
@@ -1,32 +1,25 @@
 "use strict"
 
 const express = require('express');
+const MongooseController = require('./mongooseController');
+const path = require('path');
+const routes = require('./routes');
+
 const app = express();
+const databaseAddress = (process.env.DB_ADDR || "mongodb://localhost/test");
+//const mongo = new MongooseController(databaseAddress);
```
Comment (kiosia) [inline api/server.js]: nope.avi

**Review by kiosia — CHANGES_REQUESTED** [review]

File: api/server.js (hunk 2 above)
Comment (cristiadu) [reply api/server.js]: nope.wmv

File: api/server.js (hunk 1 above)
Comment (cristiadu) [reply api/server.js]: nope-alandoni-cut.wmv

**Review by kiosia — APPROVED** [review]
lgtm

**Pull request merged**